src/main/resources/tags/info/1.18support.tag

```diff
@@ -0,0 +1,10 @@
+type: text
+aliases: 1.18
+
+---
+
+Geyser's latest release supports 1.17.1 through to 1.18.
```

Comment:
This is misleading, Geyser supports 1.12 through 1.18 but the bedrock player will be joined as 1.18. The ViaVersion part is right though.

Reply:
Oh sorry, I didn't realize that. I'll quickly fix